dashboard/installation/dashboard-migration.md

```diff
@@ -41,7 +52,20 @@ The Invictus V2 release pipeline will create a new App Service dashboard resourc
 
 **NOTE: By doing this you will lose access to the old dashboard and the ability to view and query its historic data.**
 
+## Add Role Assignments
+
+In addition to the flowhandlerjob function, in V2 also the dashboardgateway must have Logic App Contribute access on the resource group or subscription which contains the Logic Apps.
+See also [Access Control Rights](../accesscontrolrights.md)
+
+## Clear migrated AzureAD users from CosmosDB container
```

Comment:
This was probably somewhat of a bug from our end and so this step should not be required.